### ISSUE TITLE
Quick Select

### ISSUE THREAD

**Issue by ChildLearningClub** [pulsejet/memories#654]
**Is your feature request related to a problem? Please describe.**

Not really a problem as much as maybe a more user friendly way of selecting photos for sharing or downloading… often times in photo apps selection of photos is done in the “gallery” view, I propose adding in addition the option to have a quick select option or maybe even a toggle button to enable the ability to when swiping between photos in full screen to tap the screen and have the photo selected. Generally on mobile devices the thumbnails in Gallery view are just to small to see the image well enough to decide if that is the one that you want especially if there are a few very similar ones. 

**Describe the solution you'd like**

click on a photo in gallery and have it go full screen swipe left or right to go to previous or next image. If it is an image that you want to select and you have enable quick select simply tapping down on the image with the finger or thumb that you just swiped with would select that image, swipe right to go to next image quick tap of the image and now two images are selected to download share or whatever. 

**Describe alternatives you've considered**

having a button at the bottom right corner of the image in full screen to select it would also maybe be a good alternative? Would like to hear any thoughts on the idea? 

**Additional context**

**Comment by ChildLearningClub** [comment]
Above Quick Select would be combined with #472

**Comment by pulsejet** [comment]
Interesting thought, the UX might be tricky for this. Is there any other photo app (commercial?) that implements something similar for reference?

**Comment by ChildLearningClub** [comment]
Yeah was thinking about the UX swipe left and right are used up and down close out double tap used for some and the single tap used for toggling the menu options. I suppose when the quick select option is enabled the menu would be locked to visible with the additional selected/ unselected check box in the menu. The single tap would then be used for the quick select. I'm not aware of any other program that does it.

**Comment by ChildLearningClub** [comment]
oh and doesn't look like long hold when the photo/ video is full screen so that could be used to toggle/ enable the quick select the same as it is when in gallery view.

**Comment by frederikb96** [comment]
I would also be interested in a solution where you can also view photos in full screen while being in the selecting mode. It is actually already possible in the normal Nextcloud photos app. However, there it is quite basic. You open a picture by clicking on it and you select a picture by clicking on the checkbox in the top right of the picture. But there is no option to select the picture while in full screen.

I actually like that memories also selects the picture if you press somewhere on the picture, since hitting the top right corner is quite difficult.

So as far as I understand it, we have two features here that would make sense and these are possible solutions:

Feature 1: Enable full view while in select mode.
- Option 1: Do it like it is in NC photos. Disadvantage, more difficult to select, especially on mobile.
- Option 2: Do it exactly the other way around as in NC photos. Top right corner to see picture in full screen.
- Option 3: Keep it as it is but double tapping the photo enables full screen view of this picture. Good on desktop and on mobile. But maybe not intuitive. (Maybe combine with option 2)

Feature 2: Ability to select while in full screen view of a picture
- Option 1: As presented by author.
- Option 2: Alternative presented by author. Top right button to select picture. Would make sense in combination with option 2 of feature 1. (+ option 3 maybe)

I am not sure if there is a gold standard for this UI decision one should use. Maybe worth taking a look at how apple photos handles this?
